content/shared/influxdb3-plugins/plugins-library/official/basic-transformation.md

```diff
@@ -1,60 +1,70 @@
-|-----------|------|---------|-------------|
-| `window` | string | required (scheduled only) | Historical data window. Format: `<number><unit>` (for example, `"30d"`, `"1h"`) |
-| `included_fields` | string | all fields | Dot-separated list of fields to include (for example, `"temp.humidity"`) |
-| `excluded_fields` | string | none | Dot-separated list of fields to exclude |
-| `filters` | string | none | Query filters. Format: `'field:"operator value"'` |
+| Parameter         | Type   | Default                   | Description                                                          |
+|-------------------|--------|---------------------------|----------------------------------------------------------------------|
+| `window`          | string | required (scheduled only) | Historical data window. Format: `<number><unit>` (e.g., "30d", "1h") |
+| `included_fields` | string | all fields and tags       | Dot-separated list of fields and tags to include (e.g., "temp.humidity.location") |
+| `excluded_fields` | string | none                      | Dot-separated list of fields and tags to exclude                             |
+| `filters`         | string | none                      | Query filters. Format: `'field:"operator value"'`                    |
 
 ### TOML configuration
 
 | Parameter          | Type   | Default | Description                                                                      |
 |--------------------|--------|---------|----------------------------------------------------------------------------------|
 | `config_file_path` | string | none    | TOML config file path relative to `PLUGIN_DIR` (required for TOML configuration) |
 
-*To use a TOML configuration file, set the `PLUGIN_DIR` environment variable and specify the `config_file_path` in the trigger arguments.* This is in addition to the `--plugin-dir` flag when starting InfluxDB 3.
+*To use a TOML configuration file, set the `PLUGIN_DIR` environment variable and specify the `config_file_path` in the trigger arguments.* This is in addition to the `--plugin-dir` flag when starting {{% product-name %}}.
 
 #### Example TOML configurations
 
 - [basic_transformation_config_scheduler.toml](https://github.com/influxdata/influxdb3_plugins/blob/master/influxdata/basic_transformation/basic_transformation_config_scheduler.toml) - for scheduled triggers
 - [basic_transformation_config_data_writes.toml](https://github.com/influxdata/influxdb3_plugins/blob/master/influxdata/basic_transformation/basic_transformation_config_data_writes.toml) - for data write triggers
 
-```bash
-influxdb3 create trigger \
- --database mydb \
- --plugin-filename basic_transformation.py \
- --trigger-spec "every:1d" \
- --trigger-arguments config_file_path=basic_transformation_config_scheduler.toml \
- basic_transform_trigger
-```
+For more information on using TOML configuration files, see the Using TOML Configuration Files section in the [influxdb3_plugins/README.md](../../README.md).
```

Comment:
We'll need to have the script resolve/replace `(../../README.md)`, and other relative links, with the full URL. I think it should already, but didn't match here.